app/build.gradle

```diff
@@ -106,16 +106,16 @@ dependencies {
     implementation 'io.grpc:grpc-okhttp:1.21.0'
     implementation 'io.grpc:grpc-protobuf-lite:1.21.0'
     implementation 'io.grpc:grpc-stub:1.21.0'
-    implementation 'javax.annotation:javax.annotation-api:1.2'
+    implementation 'javax.annotation:javax.annotation-api:1.3.2'
     compileOnly 'com.google.api.grpc:googleapis-common-protos:0.0.3'
 
     // Used for base encoding and URI escaping
-    implementation 'com.google.guava:guava:27.0.1-android'
+    implementation 'com.google.guava:guava:28.0-android'
 
 
     implementation 'androidx.appcompat:appcompat:1.1.0-beta01'
     implementation 'com.google.android.material:material:1.1.0-alpha07'
-    implementation 'androidx.constraintlayout:constraintlayout:2.0.0-beta1'
```

Comment:
Usually I tend to only upgrade to beta versions if we are facing bugs in our app that would be solved with the new version. Otherwise I would wait for a stable release.

Reply:
Okay, seems resonable. Will do it like that next time. I didn't find any issues though after upgrading, so I'll keep it in this PR.